### PR TITLE
feat: add publish npm task

### PR DESCRIPTION
**Pull request by wa0x6e**
This PR add a new workflow, to publish a package on NPM

### CODE REVIEW

File: .github/workflows/publish-npm.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.target }}
+          registry-url: https://registry.npmjs.org/
+
+      - name: Yarn install
+        run: yarn install --frozen-lockfile
+
+      - name: Publish to NPM
+        run: npm publish
```
Comment (Todmy) [inline .github/workflows/publish-npm.yaml:27]: can we use `yarn publish` for consistency?

Comment (wa0x6e) [reply .github/workflows/publish-npm.yaml:27]: @ChaituVR what do you think ?

Comment (ChaituVR) [reply .github/workflows/publish-npm.yaml:27]: Didn't try `yarn publish` but I feel like it should work too.

**Review by Todmy — APPROVED** [review]
utACK

**Pull request merged**